app/scripts/controllers/home.js

```diff
@@ -176,6 +176,7 @@ angular.module('noteeApp').controller('textInstanceCtrl', function ($scope, $uib
       }, function errorCallback(response) {
         console.log(response);
     });
+    location.reload();
```

Comment:
Can you call $scope.loadData instead of location.reload() ? I think location.reload() will reload the whole page. After the modal is dismissed, call $scope.loadData() in HomeCtrl. Same applies to other type card modals too.